extension/contentScript.js

```diff
@@ -11,122 +11,108 @@
 // @grant        none
 // ==/UserScript==
 
-(function() {
-    'use strict';
+'use strict';
 
-    const isWatchVideoPage = () => window.location.pathname == "/watch"
+const isWatchVideoPage = () => window.location.pathname === "/watch"
```

Comment:
What's the necessity for all the triple equals?

Reply:
=== means that the type of the left-hand side is the same as the type of the right-hand side.